src/modules/mpd/mpd.cpp

```diff
@@ -144,13 +144,25 @@ void waybar::modules::MPD::setLabel() {
   bool        singleActivated = mpd_status_get_single(status_.get());
   std::string singleIcon = getOptionIcon("single", singleActivated);
 
+  auto artistLen = config_["artist-len"].isInt() ?
```

Comment:
I would prefer overriding artist with something like:
```
if (config_["artist-len"].isInt()) {
    artist = artist.substr(0, config_["artist-len"].asInt());
}
```
Rather than substr with the string length if length not provided, same for other variables.

Reply:
Yeah, that makes sense

Reply:
that makes more sense, I'll fix it up in a bit